buster/completers/base.py

```diff
@@ -171,6 +173,7 @@ def encode_df(df: pd.DataFrame) -> dict:
             "matched_documents": self.matched_documents,
             "answer_relevant": self.answer_relevant,
             "question_relevant": self.question_relevant,
+            "completion_kwargs": self.completion_kwargs,
```

Comment:
The risk I can see is if `completion_kwargs` contains non-jsonable fields.

Reply:
yes, for now it shouldnt with openai kwargs but in the future it's something to be mindful of indeed, i dont think we should worry about it right now though

Reply:
Great, looks like we are ready to merge

Reply:
@hbertrand can you see this message?

Reply:
New comment from me!